x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/GetDatafeedsStatsAction.java

```diff
@@ -62,6 +63,7 @@ private GetDatafeedsStatsAction() {
     // serialized to older nodes where the transport action was a MasterNodeReadAction.
     // TODO: Make this a simple request in a future version where there is no possibility
     // of this request being serialized to another node.
+    @UpdateForV9(owner = UpdateForV9.Owner.MACHINE_LEARNING)
```

Comment:
ML Folks: As per the comment above, can this now be updated for V9?

Reply:
👍  thanks for creating the annotation